python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1655,3 +1655,23 @@ def convert_size(node, **kwargs):
     and return the created node.
     """
     return create_basic_op_node('Size', node, kwargs)
+
+
+@mx_op.register("log_softmax")
+def convert_logsoftmax(node, **kwargs):
+    """Map MXNet's log_softmax operator attributes to onnx's LogSoftMax operator
+    and return the created node.
+    """
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
```

Comment:
what about temperature parameter?

Reply:
handled this case. ONNX doesn't support temperature, therefore, raising an Attribute error